packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```diff
@@ -19,16 +19,16 @@
   &:indeterminate {
     + .label {
       .box {
-        border-color: var(--ColorBorderChecked);
-        background-color: var(--ColorBackgroundChecked);
+        border-color: var(--CheckedBoxBorderColor);
+        background-color: var(--CheckedBoxBackgroundColor);
       }
     }
   }
 
   &:checked {
     + .label {
       .indicator {
```

Comment:
not strictly required to change the CSS class names but it is nice to do, since we're mixing `indicator` and `check` language here now

Reply:
I've swapped all of the "Check" references to "Indicator". That way we cover our Indetermine state "-" as well.